source/api_reference/index.html.md.erb

```diff
@@ -3,54 +3,58 @@ title: API reference
-- view transactions within a specified time period
-- provide full or partial refunds
+The same base URL is used for testing and live. The API key you use
+determines if the actions are treated as sandbox test payments, or processed as
+real payments.
 
-<div style="height:1px;font-size:1px;">&nbsp;</div>
-
-## API overview
-
->The base URL for the API is __https://publicapi.payments.service.gov.uk/__
-
-The same base URL is now used for testing and live. The API key you use determines if the actions are treated as sandbox test payments or processed as real payments.
+The API is based on REST principles. It returns data in JSON format, and uses
+standard HTTP error response codes. 
 
-For full details of each API action, see the API Browser:
+For full details of each API action, see the API browser: 
 
-- <a href="https://gds-payments.gelato.io/docs/versions/1.0.2/resources/general" target="blank">General functions</a> (link opens in new window)
-- <a href="https://gds-payments.gelato.io/docs/versions/1.0.2/resources/payment-id" target="blank">Payment ID functions</a> (link opens in new window)
+- <a href="https://gds-payments.gelato.io/docs/versions/1.0.2/resources/general" target="blank">General functions</a> [external link] 
+- <a href="https://gds-payments.gelato.io/docs/versions/1.0.2/resources/payment-id" target="blank">Payment ID functions</a> [external link] 
 
-You can also use our interactive <a href="https://gds-payments.gelato.io/api-explorer/gov-uk-pay-api/versions/1.0.2//" target="blank">API Explorer</a> (link opens in new window) to try out API calls and view responses.
+You can also use the interactive <a
+href="https://gds-payments.gelato.io/api-explorer/gov-uk-pay-api/versions/1.0.2//"
+target="blank">API explorer</a> [external link] to try out API calls and view
+responses.
 
-See the [Quick Start Guide](/quick_start_guide/#quick-start-guide) section for how to set up the API Explorer. Make sure you enter your sandbox API key to avoid generating real payments.
+See the ["Quick start guide"](/quick_start_guide/#quick-start-guide) section for
+how to set up the API explorer. Make sure you enter your sandbox API key to
+avoid generating real payments.
 
+## Authentication
 
+GOV.UK Pay authenticates API calls with OAuth2 HTTP bearer tokens. These are
+easy to use and contain only your API key. [RFC
+6750](http://tools.ietf.org/html/rfc6750) details bearer tokens.  
 
-## API authentication
-
-GOV.UK Pay authenticates API calls with OAuth2 HTTP bearer tokens. These are easy to use and consist of one component: your API key. Bearer tokens are specified in [RFC 6750](http://tools.ietf.org/html/rfc6750).
-
-When making an API call, you’ll need to add your API key to an "Authorization" HTTP header and prefix it with "Bearer ". This is an example of how a header would look.
+When making an API call, you need to add your API key to an "Authorization"
+HTTP header prefixed with "Bearer ". For example: 
```

Comment:
I think this wording is a little ambiguous about what gets the prefix added to it (the old wording had “it”, which clearer referred to the API key).

If it helps to come up with better wording, `Authorization` is the header (field) name, `Bearer <YOUR-API-KEY-HERE>` is the header (field) value and `Authorization: Bearer <YOUR-API-KEY-HERE>` all together is header (field).